src/relative-time/properties.js

```diff
@@ -0,0 +1,41 @@
+define(function() {
+
+/**
+ * properties( unit, cldr, options )
+ *
+ * @unit [String] eg. "day", "week", "month", etc.
+ *
+ * @cldr [Cldr instance].
+ *
+ * @options [Object]
+ * - form: [String] eg. "short" or "narrow". Or falsy for default long form
+ *
+ * Return relative time properties.
+ */
+return function( unit, cldr, options ) {
+
+	var form = options.form,
+		raw, properties, key, match;
+
+	if ( form ) {
+		unit = unit + "-" + form;
+	}
+
+	raw = cldr.main( [ "dates", "fields", unit ] );
+	properties = {
+		"relativeTime-type-future": raw[ "relativeTime-type-future" ],
+		"relativeTime-type-past": raw[ "relativeTime-type-past" ]
+	};
+	for ( key in raw ) {
+		if ( raw.hasOwnProperty( key ) ) {
+			match = /relative-type-(-?[0-9]+)/.exec( key );
+			if ( match ) {
+				properties[ key ] = raw[ key ];
+			}
+		}
+	}
```

Comment:
Given we've dropped `maximumWordOffset` and the overall logic is simpler, we can simplify the code with:

```diff
-	raw = cldr.main( [ "dates", "fields", unit ] );
-	properties = {
-		"relativeTime-type-future": raw[ "relativeTime-type-future" ],
-		"relativeTime-type-past": raw[ "relativeTime-type-past" ]
-	};
-	for ( key in raw ) {
-		if ( raw.hasOwnProperty( key ) ) {
-			match = /relative-type-(-?[0-9]+)/.exec( key );
-			if ( match ) {
-				properties[ key ] = raw[ key ];
-			}
-		}
-	}
-
-	return properties;
+	return objectFilter(
+		cldr.main([ "dates/fields", unit ]),
+		/relativeTime-type-(future|past)|relative-type--?[0-9]+/
+	);
```

Note you'll need to:
```diff
-define(function() {
+define([
+	"../util/object/filter"
+], function( objectFilter ) {
```